tests/foreman/ui/test_repository.py

```diff
@@ -714,6 +715,33 @@ def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+def test_positive_no_errors_on_repo_scan(session, default_sat):
+    """Scan repos for RHEL Server Extras, then check the production log
+    for a specific error
+
+    :id: 443bf4af-7f9a-48b8-8f98-fdb170e8ae88
+
+    :expectedresults: The specific error isn't contained in the prod log
+
+    :BZ: 1994212
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    manifests.upload_manifest_locked(org.id)
+    sat_rpm_extras = RHELServerExtras(cdn=True)
+    with session:
+        session.organization.select(org.name)
+        session.redhatrepository.read(sat_rpm_extras.data['repository-set'])
+        result = default_sat.execute(
+            'cat /var/log/foreman/production.log | '
+            'rep "Failed at scanning for repository: undefined method '
```

Comment:
bash: rep: command not found
Did you mean `grep`? If so, could you try drop the use of cat and just the grep or even just assert the error is not in the log?
BTW, in the logging tests[1] there are some useful examples of cutting log files using the `cut_lines` helper.[2].
[1] https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_logging.py
[2] https://github.com/SatelliteQE/robottelo/blob/cbb6ae34dee15b8fc868a3c3af4560af7c3e9a61/robottelo/helpers.py#L138